builder/static_code.go

```diff
@@ -221,7 +221,7 @@ type position struct {
 }
 
 func (p position) String() string {
-	return fmt.Sprintf("%d:%d [%d]", p.line, p.col, p.offset)
+	return strconv.Itoa(p.line) + ":" + strconv.Itoa(p.col) + " [" + strconv.Itoa(p.offset) + "]"
```

Comment:
Have you tested if the usage of a `strings.Builder` (or even `bytes.Buffer`) increases the performance even more?